js/emoji_util.js

```diff
@@ -46,6 +120,7 @@
         if (!$el || !$el.length) {
             return;
         }
+
         $el.html(emoji.replace_unified($el.html()));
```

Comment:
🤔 It occurs to me that we could just post-process the converted output with jquery rather than digging (further*) into the guts of EmojiConvertor.
```
$emojis = $el.find('img.emoji');
emojiCount = emojis.length;
...
emojis.addClass(...)
```

However, according to git blame, I decided to go down this road at some point, so it's possible past me knew something that current me is forgetting about. :woman_shrugging: